src/oph/ehoks/palaute/scheduler.clj

```diff
@@ -0,0 +1,31 @@
+(ns oph.ehoks.palaute.scheduler
+  (:require [chime.core :as chime]
+            [clojure.tools.logging :as log]
+            [oph.ehoks.palaute.tyoelama :as tep])
+  (:import (java.lang AutoCloseable)))
+
+(defn- run-sequence
```

Comment:
Näille toiminnoille ei varmaan oo vielä APIa?  

Toive: tehdään tai tiketöidään.  Helpottaa käsin testaamista, myös tämän PR:n

Reply:
On jo tiketöity: https://jira.eduuni.fi/browse/EH-1701

Reply:
Joo sorry että sanoin tästä näin paljon, unohdin että olin puhunut siitä jo aiemmin moneen kertaan :D